test/jdk/java/net/SocketOption/options.policy

```diff
@@ -26,3 +26,11 @@ grant {
     permission java.util.PropertyPermission "java.net.preferIPv4Stack", "read";
     permission java.util.PropertyPermission "java.net.preferIPv6Addresses", "read";
 };
+// for JTwork/classes/test/lib/jdk/test/lib/Platform.class
+grant codeBase "file:${test.classes}/../../../../test/lib/-" {
+    permission java.util.PropertyPermission "*", "read";
+};
+// for JTwork/classes/test/lib/jdk/test/lib/NetworkConfiguration.class
+grant codeBase "file:${test.classes}/../../../../test/lib/-" {
```

Comment:
A single grant clause with the two permissions should be enough - since both classes are in the same codebase. Also I thought the plan was to grant `SocketPermission "*:*" ...` to the test? If so then you could simply add the two permissions to the global grant clause. No need to distinguish between the permissions granted to the library and the permissions granted to the test.

Reply:
I modeled this after other tests which explicitly grant permissions to the library code. But you're right, we don't need to distinguish between these permissions; the SM test was introduced by JDK-8245828, and the new permissions will not interfere with its original purpose.